docs/tiles.html

```diff
@@ -165,192 +297,53 @@ <h2>Forms for advanced users</h2>
-                </a>
-                <a transform="rotate(180 127.07 84.546)" style="fill:#000000" xlink:href="javascript:swDown()">
-                    <path d="m85.252 41.366-15.094 0.004029 7.5434-13.073z" style="fill:#000000;opacity:.883;stroke-linejoin:round;stroke-width:.9375;stroke:#ffffff"/>
-                </a>
-                <a transform="rotate(-90 152.41 45.218)" style="fill:#000000" xlink:href="javascript:swToLeft()">
-                    <path d="m95.085 41.366-15.094 0.004029 7.5434-13.073z" style="fill:#000000;opacity:.883;stroke-linejoin:round;stroke-width:.9375;stroke:#ffffff"/>
-                </a>
-                <rect x="153.77" y="77.233" width="45.35" height="45.35" ry=".48922" style="fill:none;opacity:.883;stroke-linecap:round;stroke-width:4.5094;stroke:#029807"/>
-                <rect x="220.57" y="77.233" width="45.35" height="45.35" ry=".48922" style="fill:none;stroke-linecap:round;stroke-width:4.5094;stroke:#834bce"/>
-                <path d="m33.025 59.779h-5.2874l2.6437 4.579z" style="fill:#029807"/>
-                <path d="m61.309 58.361-3.7387 3.7387 5.1072 1.3684z" style="fill:#834bce;stroke-width:1.2031"/>
-                <rect x="19.22" y="110.85" width="27.529" height="12.467" ry=".13449" style="fill:none;stroke-linecap:round;stroke-width:2.3438;stroke:#029807"/>
-            </svg>
-            <p style="max-width: 22em">
-			Click a solid tile to initialize the prototype,
-            subsequently nudge the tiles with the arrows.
-		</p>
-        <div>
-            <div>Configuration:</div><br>
-            <div>
-                <div style="border:3px solid #029807;border-radius: 3px;">
-                    <input type="number" name="shiftColsSW" id="shiftColsSW" min="-12" max="0" value="0" onchange="showProto()"><label for="shiftColsSW">col(s)</label>
-                    <br>
-                    <input type="number" name="shiftRowsSW" id="shiftRowsSW" min="0" max="12" value="1" onchange="showProto()"><label for="shiftRowsSW">row(s)</label>
-                </div>
-                <div style="border:3px solid #834bce;border-radius: 3px;">
-                    <input type="number" name="shiftColsSE" id="shiftColsSE" min="0" max="12" value="3" onchange="showProto()"><label for="shiftColsSE">col(s)</label>
-                    <br>
-                    <input type="number" name="shiftRowsSE" id="shiftRowsSE" min="0" max="12" value="1" onchange="showProto()"><label for="shiftRowsSE">row(s)</label>
-                </div>
-            </div>
+        <div id="drosteFields3">
+            <figure>
+                <figcaption>
+                    Stitches
+                    <a href="help/Advanced#thread-diagram-as-pair-diagram" target="_blank"><img src="help/images/information-icon.png" title="help"></a>
+                </figcaption>
+                <textarea id="droste3" name="droste3" rows="7" cols="30" spellcheck="false" onchange="clear3()">ct&#13;&#10;a140=a150=b240=b250=ctcctct</textarea>
```

Comment:
what I meant was 
```suggestion
                <textarea id="droste3" name="droste3" rows="7" cols="30" spellcheck="false" onchange="clear3()" placeHolder="ct&#13;&#10;a140=a150=b240=b250=ctcctct"></textarea>
```
It works like you often see a grey `https://` or some fake email address in form fields that disappears as soon as you start typing.